dot/network/host.go

```diff
@@ -368,6 +368,68 @@ func (h *host) peers() []peer.ID {
 	return h.h.Network().Peers()
 }
 
+// addReservedPeers adds the peers `addrs` in the Peerstore
+func (h *host) addReservedPeers(addrs ...string) error {
+	if len(addrs) < 1 {
+		return nil
+	}
+
+	for _, addr := range addrs {
+		maddr, err := ma.NewMultiaddr(addr)
+		if err != nil {
+			return err
+		}
+
+		addinfo, err := peer.AddrInfoFromP2pAddr(maddr)
+		if err != nil {
+			return err
+		}
+
+		h.h.Peerstore().AddAddrs(addinfo.ID, addinfo.Addrs, peerstore.PermanentAddrTTL)
```

Comment:
```suggestion
		```suggestion
         // you'll need to add the `ConnManager` to the host
		h.connManager.persistentPeers.Store(addinfo.ID)
		err := h.connect(addinfo)
		// handle err
```
```